lib/ClangImporter/ImportDecl.cpp

```diff
@@ -7903,9 +7903,39 @@ createUnavailableDecl(Identifier name, DeclContext *dc, Type type,
   return var;
 }
 
+static ExtensionDecl *figureOutTheDeclarationContextToImportInto(Decl* D, DeclContext *&DC, IterableDeclContext *&IDC) {
+  if (auto nominal = dyn_cast<NominalTypeDecl>(D)) {
+    DC = nominal;
+    IDC = nominal;
+    return nullptr;
```

Comment:
Also, in LLVm style I think the else is on the same line as the }. git-clang-format I think will take care of that for you.